tests/Doctrine/Tests/ORM/ConfigurationTest.php

```diff
@@ -179,6 +179,8 @@ public function testEnsureProductionSettings()
     {
         $this->setProductionSettings();
         $this->configuration->ensureProductionSettings();
+
+        $this->addToAssertionCount(1);
```

Comment:
Mmmm sorry. I am looking for any documentation about this addToAssertionCount. Just found something grepping phpunit. "addToAssertionCount" is just a "technique" to avoid phpunit debug messages of test that has no assertion?